clients/admin-ui/src/features/system/add-multiple-systems/MultipleSystemsFilterModal.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  Divider,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@fidesui/react";
+import { Table as TableInstance } from "@tanstack/react-table";
+import React, { useMemo, useState } from "react";
+
+type FilterCheckboxProps = {
+  onChange: () => void;
+  displayText: string;
+  value: string;
+  isChecked: boolean;
+};
+
+const FilterCheckbox = ({
+  onChange,
+  displayText,
+  isChecked,
+  value,
+}: FilterCheckboxProps) => (
+  <Checkbox
+    value={value}
+    key={value}
+    height="20px"
+    mb={3}
+    isChecked={isChecked}
+    onChange={onChange}
+    colorScheme="complimentary"
+    mr={5}
+  >
+    <Text
+      fontSize="sm"
+      lineHeight={5}
+      height="20px"
+      textOverflow="ellipsis"
+      overflow="hidden"
+      whiteSpace="nowrap"
+    >
+      {displayText}
+    </Text>
+  </Checkbox>
+);
+
+interface MultipleSystemsFilterProps<T> {
+  tableInstance: TableInstance<T>;
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const initialFilterState = {
+  gvl: false,
```

Comment:
is it possible to use that new `VendorSources` enum here?

Reply:
Possibly, I'll take a look

Reply:
It is possible. great idea!

https://github.com/ethyca/fides/pull/4351/commits/461014b8b5b5ee31257133e32cccdf9cd9fa1f62